fastlane/lib/fastlane/helper/sh_helper.rb

```diff
@@ -68,5 +68,15 @@ def self.sh_control_output(command, print_command: true, print_command_output: t
       Encoding.default_external = previous_encoding.first
       Encoding.default_internal = previous_encoding.last
     end
+
+    def self.command_from_args(*args)
```

Comment:
Some nitpicking: maybe rename this from `command_from_args` to `shell_command_from_args`. Even though it is in the shell_helper file it can be called from outside via `Fastlane::Actions` so this should make it more explicit what is happening